### DISCUSSION TITLE
settings in discussons - to appear on the github-page - doable!?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#36185] [Discussions]
dear friends 

i have had a quick view onto the documents after setting up the dicussions - 
https://resources.github.com/devops/process/planning/discussions/

what is aimed : i have set up the discussion here: 
the Discussions-Page: https://github.com/fsj-digital/DigitalHub/discussions/1

am i able to add the discussions somehow on the page here: https://fsj-digital.github.io/DigitalHub/

in other words: is it possible to set the discussons - to appear on the github-page - doable!?

**Comment by jfullstackdev** [comment]
that cannot be seen by others, I think so, 

this is on my end 

![image](https://user-images.githubusercontent.com/47092464/196042360-0603b539-f8cb-4bab-a337-c7464ee1c380.png)

if you have another site, which is not GitHub, maybe you can embed the discussion there, so it will appear on your other site, I'm not just very sure if GitHub will allow that, not all sites can be embedded
